apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryRow.cs

```diff
@@ -169,6 +171,8 @@ private static object ConvertSingleValue(object rawValue, TableFieldSchema field
                     return GeographyConverter((string) rawValue);
                 case BigQueryDbType.Struct:
                     return ConvertRecord((JObject)rawValue, field);
+                case BigQueryDbType.Json:
```

Comment:
Include this in the BigQueryDbType.String case?

(In a separate commit, we could convert this to a switch expression, too. Don't do it in this PR though. I suspect a search for "switch" in the BQ libraries will give us lots of them... I'm happy to do that another time.)

Reply:
Thanks. Fixed.

Sure Jon. I can raise a new PR with with conversion to switch expression early next week.